lib/persisted-model.js

```diff
@@ -1155,6 +1156,11 @@ module.exports = function(registry) {
     var Change = sourceModel.getChangeModel();
     var TargetChange = targetModel.getChangeModel();
     var changeTrackingEnabled = Change && TargetChange;
+    var chunkSize = CHUNK_SIZE;
+
+    if (this.settings && this.settings.chunkSize) {
```

Comment:
I believe `this` is `undefined` in this function. I think you should be using `sourceModel.chunkSize` instead?

It would be great to have a unit-test verifying this setting - if there was such test, then the problem with `this` would have been discovered long time ago.

I also find the name `chunkSize` as not descriptive enough. This setting will be usually configured in model JSON file, where it will "sit" among other general settings and it won't be obvious that `chunkSize` is related to change replication.

I am proposing `replicationChunkSize`, but feel free to come up with a different name that makes it clear that the chunk size refers to change replication/sync.